Update Get-AzStorageBlob.md (#26112)
Azure/azure-powershell: src/Storage/Storage.Management/help/Get-AzStorageBlob.md

This command gets a single blob with blob tag condition. 
The cmdlet will only success when the blob contains a tag with name "tag1" and value "value1", else the cmdlet will fail with error code 412.

> ### Example 10: Get blob properties

Reviewer: How about this name:

Get blob properties (ImmutabilityPolicy) on single blob

> ### Example 10: Get blob properties

Reviewer: ```suggestion
### Example 10: Get blob properties (ImmutabilityPolicy) on single blob
```

```powershell

> (Get-AzStorageBlob -Container "ContainerName" -Blob "blob" -Context $ctx).BlobProperties.ImmutabilityPolicy

Reviewer: How about first get the blob properties, then show the ImmutabilityPolicy.
This is useful when user want to get multiple properties of the single blob.
```suggestion
$blobProperties = (Get-AzStorageBlob -Container "ContainerName" -Blob "blob" -Context $ctx).BlobProperties
$blobProperties.ImmutabilityPolicy
```

```

```output
ExpiresOn                   PolicyMode
---------                   ----------
9/17/2024 2:49:32 AM +00:00   Unlocked
```

> This example command gets the immutability property of a single blob. BlobProperties cmdlet gets a detailed list of properties for a single blob.

Reviewer: The "BlobProperties" and "ListBlobProperties" are not cmdlet, but child properties of the output of "Get-AzStorageBlob" cmdlet.
```suggestion
This example command gets the immutability property of a single blob. You can get a detailed list of this blob properties from the **BlobProperties** property.
```

> For a list of multiple blobs, use the cmdlet without blob name and use **ListBlobProperties** cmdlet for a better performance.

Reviewer: The "BlobProperties" and "ListBlobProperties" are not cmdlet, but child properties of the output of "Get-AzStorageBlob" cmdlet.
```suggestion
For list multiple blobs (execute the cmdlet without blob name), use **ListBlobProperties** instead of **BlobProperties** for a better performance.
```

### CommonParameters
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).

> ## BlobProperties

Author: Hello @blueww , made those changes now to the best of my understanding. Please feel free to comment or modify as needed.
